go/engine/pgp_export_key.go

```diff
@@ -166,6 +181,49 @@ func (e *PGPKeyExportEngine) exportSecret(ctx *Context) error {
 	return nil
 }
 
+func getPGPPassphrase(g *libkb.GlobalContext, ui libkb.SecretUI) (keybase1.GetPassphraseRes, error) {
+	desc := "Enter passphrase to protect your PGP key. Secure passphrase should have at least 8 characters."
+	pRes, err := libkb.GetSecret(g, ui, "PGP key passphrase", desc, "", false)
+	if err != nil {
+		return keybase1.GetPassphraseRes{}, err
+	}
+
+	desc = "Please reenter your passphrase for confirmation"
```

Comment:
the reconfirm doesn't seem necessary.  If they messed it up, they can always reexport?